app/views/AcceptanceDateView.scala.html

```diff
@@ -0,0 +1,45 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(layout: templates.Layout,
+        formHelper: FormWithCSRF,
+        errorSummary: components.errorSummary,
+        yesNoRadio: components.yesNoRadio,
+        button: components.button)
+
+@(form: Form[_])(implicit request: Request[_], messages: Messages)
+
+    @layout(pageTitle = messages("acceptanceDate.pageTitle"), form = Some(form), customBackLinkUrl = Some(controllers.routes.EntryDetailsController.onLoad)) {
+
+        @errorSummary(form.errors)
+
+        @formHelper(action = controllers.routes.AcceptanceDateController.onSubmit()) {
+            @yesNoRadio(
+                form,
+                headingKey = "acceptanceDate.heading",
+                inline = false,
+                yesRadioMsg = "acceptanceDate.yes",
```

Comment:
Why are you overriding the default Yes No descriptions?

Reply:
removed yesRadioMsg and noRadioMsg